src/corner_kick/src/utils/ros/index.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * This file contains ROS methods to get and set values in the ROS Param Server
+ */
+
+import { Message, Ros, Service, ServiceRequest, Topic } from 'roslib';
+
+const ros: Ros = new Ros({});
+
+/**
+ * Connect to ROS
```

Comment:
More detailed comment describing each param (using @params, see JSDoc format) would be nice. Also indicate under what condition the Promise is accepted vs rejected.

Reply:
Updated to describe each param.